plugins/producer_plugin/producer_plugin.cpp

```diff
@@ -1098,9 +1171,8 @@ class producer_plugin_impl : public std::enable_shared_from_this<producer_plugin
    void resume() {
       _pause_production = false;
       // reset vote received so production can be explicitly resumed, will pause again when received vote time limit hit again
-      auto now = fc::time_point::now();
-      _last_other_vote_received.store(now, std::memory_order_relaxed);
-      _last_producer_vote_received.store(now, std::memory_order_relaxed);
+      if (_is_savanna_active)
+         _implicit_pause_vote_tracker.force_unpause();
```

Comment:
Guarding this with an if statement is unnecessary (though also it does no harm to keep it as is).